comment/api/serializers.py

```diff
@@ -22,10 +25,14 @@ def get_profile_model():
 def get_user_fields():
     user_model = get_user_model()
     fields = user_model._meta.get_fields()
+    api_fields = list(settings.COMMENT_USER_API_FIELDS)
+    api_fields.append('profile')
+    # remove duplicates while preseving order
+    api_fields = list(OrderedDict.fromkeys(api_fields))
     for field in fields:
-        if hasattr(field, "upload_to"):
-            return 'id', 'username', 'email', 'profile', field.name
-    return 'id', 'username', 'email', 'profile'
+        if isinstance(field, ImageField) and hasattr(field, "upload_to"):
+            api_fields.append(field.name)
+    return list(api_fields)
```

Comment:
`api_fields ` is already list you don't need to convert it to list again? 
```suggestion
    return api_fields
```